recipes/stack_data/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "stack_data" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: df93061c669cba1a2256689c7fb19603be99f96d46a107dc5306c11be847ea3c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  skip: true  # [py<35]
```

Comment:
I think this recipe can be `noarch`. In that case, please remove the `skip` line, and use `python >=3.5` in both `host` and `run` sections below.

Reply:
Here is some information about `noarch` packages, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages.